Bottom: 05a9830fce9cddc1866764d58df23c5e2ae1c7ea
Top:    9b63ed6c3e8bdda0da38291bde90cabb41d5d1d1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-01-14 17:32:23 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
